test/projection.jl

```diff
@@ -69,6 +69,10 @@ Base.zero(x::Dual) = Dual(zero(x.value), zero(x.partial))
         @test prow(transpose([1, 2, 3 + 4.0im])) isa Matrix  # row vectors may not pass through 
         @test prow(adjoint([1, 2, 3 + 5im])) == [1 2 3 - 5im]
         @test prow(adjoint([1, 2, 3])) isa Matrix
+
+        # some bugs
+        @test pvec3(fill(NoTangent(), 3)) === NoTangent() #410, was an array of such
+        @test ProjectTo([pi])([1]) isa Vector{Int} #423, was Irrational -> Bool -> NoTangent
```

Comment:
BlueStyle likes 2 spaces before end of line comments
```suggestion
        @test pvec3(fill(NoTangent(), 3)) === NoTangent()  #410, was an array of such
        @test ProjectTo([pi])([1]) isa Vector{Int}  #423, was Irrational -> Bool -> NoTangent
```

Reply:
Does it have opinions on overloading the same # to be a comment and an issue number? :)